app/assets/javascripts/discourse/app/helpers/category-link.js

```diff
@@ -185,11 +185,5 @@ export function defaultCategoryLinkRenderer(category, opts) {
   if (opts.topicCount && categoryStyle === "box") {
     afterBadgeWrapper += buildTopicCount(opts.topicCount);
   }
-  if (opts.plusSubcategories && opts.lastSubcategory) {
```

Comment:
Why was this deleted?

Reply:
It was such an obscure feature I doubt many people used. Then, it was hard to implement with the new async search model. I can add it back in the future if people want it back.